hera_cal/io.py

```diff
@@ -984,9 +987,9 @@ def partial_write(self, output_path, data=None, flags=None, nsamples=None,
 
         # else:  # make a copy of this object and then update the relevant arrays using DataContainers
         #     this = copy.deepcopy(self)
-        
-        hd_writer.write_uvh5_part(output_path, d, f, n, 
-                                  run_check_acceptability=output_path in self._writers, 
+
+        hd_writer.write_uvh5_part(output_path, d, f, n,
+                                  run_check_acceptability=output_path in self._writers,
```

Comment:
I'm confused about setting `run_check_acceptability=output_path`, I think `run_check_acceptability` should be a boolean.

Reply:
it's `run_check_acceptability=output_path in self._writers`, which isn't wrong but would be clearer with parentheses. Added those.